target_chains/solana/sdk/js/pyth_solana_receiver/src/PythSolanaReceiver.ts

```diff
@@ -147,11 +166,50 @@ export class PythTransactionBuilder extends TransactionBuilder {
     this.addInstructions(postInstructions);
   }
 
+  /**
+   * Add instructions to update price feed accounts to the builder.
+   *
+   * @param priceUpdateDataArray the output of the `@pythnetwork/price-service-client`'s `PriceServiceConnection.getLatestVaas`. This is an array of verifiable price updates.
+   * @param shardId the shard ID of the set of price feed accounts. This shard ID allows for multiple sets of price feed accounts to be managed by the same program.
+   *
+   * Price feed accounts are a special type of price update accounts.
+   * Instead of using ephemeral addresses, they are PDAs of the Pyth Push Oracle program derived from the feed ID. They can only be updated with a more recent price update.
+   *
+   * @example
+   * ```typescript
+   * const priceUpdateData = await priceServiceConnection.getLatestVaas([
+   *    SOL_PRICE_FEED_ID,
+   *    ETH_PRICE_FEED_ID,
+   * ]);
+   *
+   * const transactionBuilder = pythSolanaReceiver.newTransactionBuilder({});
+   * await transactionBuilder.addUpdatePriceFeed(priceUpdateData);
+   * await transactionBuilder.addPriceConsumerInstructions(...)
+   * ...
+   * ```
+   */
+  async addUpdatePriceFeed(priceUpdateDataArray: string[], shardId: number) {
```

Comment:
Is there anywhere (like in the readme) that we explain the difference between addPostPriceFeed vs addUpdatePriceFeed? I think it can get confusing to users.